README.md

```diff
@@ -69,6 +69,30 @@ Go to `Cursor Settings` -> `MCP` -> `Add new MCP Server`. Name to your liking, u
 
 </details>
 
+<details>
```

Comment:
Let's try to preserve the alphabetic order.

Reply:
Ah, didn't notice that 😅 . Fixed!